index.html

```diff
@@ -215,19 +221,19 @@ <h3>Skills</h3>
         </section>
     </main>
     <footer>
-        <div class="bottom" id="f">
-            <h2 class="white">Contact me </h2>
-            <p class="call">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-            <form action="https://formspree.io/f/xkneznwb"
-            method="POST">
-    
-                <input type="text" id="fname" name="name" placeholder="Your Name" value="John" maxlength="30" required><br>
-                <input type="email" id="lname" name="email" placeholder="Your Email" required value="Doe@gmail.com" ><br>
-                <textarea name="message" placeholder="Write your message here" required maxlength="500"></textarea></br>
-                <input type="submit" value="Get in Touch" class="submit">
-              </form>
+        <div class="bottom">
+          <h2 class="white">Contact me</h2>
+          <p class="call">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          <form id="contact-form" action="https://formspree.io/f/xkneznwb" method="POST">
+            <input type="text" id="fname" name="name" placeholder="Your Name" value="John" maxlength="30" required><br>
+            <input type="email" id="email" name="email" placeholder="Your Email" required value="Doe@gmail.com" ><br>
+            <textarea name="message" placeholder="Write your message here" required maxlength="500"></textarea><br>
```

Comment:
- [ ] `required`
- [ ] 👋  please delete the values `value="John` and ` required value=Doe@gmail.com` because they are affecting the placeholder instead of the user to type in name and email, they will have to delete the required values. please make amends 🥂

Reply:
It was done.  Thank you for your feedback.